### PR TITLE
[ADD] gitlab: Add example to use gitlab

### PR DESCRIPTION
**Pull request by moylop260**
pipeline vauxoo instance gitlab 1

### CODE REVIEW

**Comment by moylop260** [comment]
@Yajo 
FYI I have added a example to use gitlab with MQT using the current scripts of runbot, travis and another CI (shippable maybe).

**Comment by nhomar** [comment]
+1

File: sample_files/.gitlab-ci.yml [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+
+test:
+  stage: test
+  tags:
+    - test
+  variables:
+    LINT_CHECK: "0"
+    TESTS: "1"
+  script:
+    - export TRAVIS_BUILD_DIR=$(pwd)
```
Comment (Yajo) [inline sample_files/.gitlab-ci.yml]: I think this can get into the main `variables` section with just `TRAVIS_BUILD_DIR: "$CI_PROJECT_DIR"`

Comment (moylop260) [reply sample_files/.gitlab-ci.yml]: I got it
Thanks!

Done!

cc @ruiztulio

File: sample_files/.gitlab-ci.yml [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+  stage: test
+  tags:
+    - test
+  variables:
+    LINT_CHECK: "0"
+    TESTS: "1"
+  script:
+    - apt-get update && apt-get install -y tree dos2unix
+    - source /.repo_requirements/virtualenv/python3.5/bin/activate
+    - git clone https://github.com/vauxoo/maintainer-quality-tools.git -b master ${HOME}/maintainer-quality-tools
```
Comment (lasley) [inline sample_files/.gitlab-ci.yml]: And here

File: sample_files/.gitlab-ci.yml [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+lint:
+  stage: test
+  tags:
+    - lint
+  variables:
+    LINT_CHECK: "1"
+    TESTS: "0"
+  script:
+    - sudo apt-get update && sudo apt-get install -y dos2unix
+    - git clone https://github.com/vauxoo/maintainer-quality-tools.git -b master ${HOME}/maintainer-quality-tools
```
Comment (lasley) [inline sample_files/.gitlab-ci.yml]: IMO this should be the OCA repo instead of Vauxoo fork

Comment (moylop260) [reply sample_files/.gitlab-ci.yml]: Good point, thanks
(copy&paste issue)

**Pull request merged**